app/services/manager.py

```diff
@@ -0,0 +1,18 @@
+from app.models import User
+from app.config.db_config import SessionLocal
+
+def add_employee_as_manager(data):
```

Comment:
What is the use case of this function, We already have an add_employee function, Where you can add role as employee by default but if we have role field in request body then we can add the user as manager.